tools/parse-ramdisk-logs.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import sys
+
+# extraction datetime is always irrelevant
+EXTRACTION_TS = re.compile(r"^[0-9TZ\.:\-]* ?")
+CONTENTS = re.compile(r"\*{12} Contents of (?P<path>/.*)\.tar\.gz ")
+ENTRY = re.compile(r"\*{4} Entry: (?P<entry>.*) \*{4}")
+
+
+def log(msg):
+    print(msg, file=sys.stderr)
+
+
+parser = argparse.ArgumentParser()
```

Comment:
I'd put this in a function that is called from main, but if you don't wish to pass args around, its fine.